### PR TITLE
Added Github Workflow for Greeting contributor on their first PR merge 

### LINKED ISSUE #411: Feature to greet users when they merge their first PR
**Summary**- I would like to implement a feature that automatically greets new users when they open their first pull request or issue in our repository. This will help us foster a welcoming and supportive community environment.

Expected Behavior -
Upon opening their first pull request or issue, a new user should receive a friendly greeting message automatically.

Additional Information -
Use Case: This feature will enhance the onboarding experience for new contributors and reinforce a positive community atmosphere.
Impact: We anticipate that this feature will help increase user engagement and foster a sense of belonging within our community.

Expected outcomes after the implementation:

Example 1
```
🎉 Welcome, [@username](link_to_user_profile)! 🎉

Thank you for opening your first pull request! We're excited to see your contribution. If you have any questions or need assistance, feel free to ask. Happy coding! 😊
```

Example 2
```
🚀 Hey there, [@username](link_to_user_profile)!

Congratulations on making your first pull request! Your initiative is appreciated. Don't hesitate to explore more areas of the project, and remember, every contribution counts! 💪
```

### PR DESCRIPTION
**Pull request by Maykkkk**
**Issue Number:**
-  Closes #411

### CODE REVIEW

File: components/Layout.tsx [hunk 1]
```diff
@@ -174,11 +174,13 @@ const MainNavigation = () => {
   )
 }
 
-const MobileNav = () => {
+const MobileNav: React.FC = () => {
```
Comment (benjagm) [inline components/Layout.tsx:177]: Why adding this modification in this file in this pr for a github workflow? Lets remove this file from it.

File: .github/workflows/greet_on_first_merge.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+              pull_requests: {
+                state: 'closed',
+                author: authorLogin,
+              },
+            });
+            
+            console.log(firstPR.data.length); 
+
+            if (firstPR.data.length === 1) {
+              const greetingMessage = ` Congratulations, @${authorLogin}! This is your first pull request merge. Thank you for your contribution! `;
```
Comment (benjagm) [inline .github/workflows/greet_on_first_merge.yml:30]:
```suggestion
              const greetingMessage = ` Congratulations, @${authorLogin} for your first pull request merge in this repository! 🎉🎉. Thanks for your contribution to JSON Schema! `;
```

**Review by benjagm — APPROVED** [review]
Great work with workflow! The code is clean and does exactly what we expect it to do.

Looks great! Just left a small suggestion with the greeting message.

File: components/Layout.tsx (hunk 1 above)
Comment (Maykkkk) [reply components/Layout.tsx]: I think it somehow came up with git push there are no changes in this file required. I have resolved this issue. Please have a look again.

**Review by benjagm — APPROVED** [review]
Great job!

**Pull request merged**